api/lib/firewall_functions

```diff
@@ -279,3 +279,60 @@ function fw_rule
         success
     fi
 }
+
+#
+# Create or update a firewall rule
+# The rule is passed by $data global variable
+# Takes one argument: action to perform (create/update)
+#
+# Usage example: fw_rule create
+#
+
+function fw_separator
+{
+    if [ "$1" = "create" ]; then
+        id=$(/usr/libexec/nethserver/api/nethserver-firewall-base/lib/rules-next-id)
+    elif [ "$1" = "update" ]; then
+        id=$(_get id)
+    else
+        error
+    fi
+    
+    Dst=$(_get Dst)
+
+    /sbin/e-smith/db separators set "$id" separator \
+        Position "$(_get Position)"  \
+        color "$(_get color)"  \
+        Dst "$Dst" \
```

Comment:
Just remove the `Dst` field, we are going to have separators only on rules page.

Reply:
done